api/src/main/java/org/openmrs/validator/UserValidator.java

```diff
@@ -126,17 +126,20 @@ public void validate(Object obj, Errors errors) {
 			}
 			
 			
-			if (!StringUtils.isBlank(user.getEmail())) {
-				if(!isEmailValid(user.getEmail())) {
+			if (!StringUtils.isBlank(user.getEmail()) && !isEmailValid(user.getEmail())) {
+
 					errors.rejectValue("email", "error.email.invalid");
 				}
-			}
 			
+			 else {
```

Comment:
Try to stick to the rule that "else" is on the same line as the earlier brace.

Reply:
Finished thanks